common/lib/xmodule/xmodule/modulestore/mongo/draft.py

```diff
@@ -830,6 +836,47 @@ def _verify_branch_setting(self, expected_branch_setting):
                 actual_setting=actual_branch_setting
             )
 
+    def _format_course_structure(self, course_key, structure):
+        course_block_key = unicode(BlockUsageLocator(course_key, 'course', course_key.run))
+        structure[course_block_key] = {
+            u'display_name': self.get_course(course_key).display_name,
```

Comment:
Oh, sorry. Misread what you were doing here. You want `depth=0`, which I think is the default.